src/main/kotlin/com/nextstep/racingcar/RacingCarApplication.kt

```diff
@@ -10,18 +10,33 @@ object RacingCarApplication {
     fun main(args: Array<String>) {
         val racingGame = generateRacingGame()
         OutputView.showResult()
-        racingGame.runGame()
-        racingGame.findWinners()
+
+        for (i in 1..racingGame.round) {
+            racingGame.runRound()
+            OutputView.showStatus(racingGame.cars)
+        }
```

Comment:
반복문 대신 repeat()을 사용해보세요.

```suggestion
        repeat(racingGame.round) {
            racingGame.runRound()
            OutputView.showStatus(racingGame.cars)
        }
```